zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -1221,36 +1221,23 @@ protected Observer makeObserver(FileTxnSnapLog logFactory) throws IOException {
     }
 
     @SuppressWarnings("deprecation")
-    protected Election createElectionAlgorithm(int electionAlgorithm) {
```

Comment:
why remove this entirely? Wouldn't it be a good idea to keep it available for folks interested in implementing their own?

Reply:
I agree with @phunt . Let's keep this patch as simple as possible which would also make backporting easier. I would just blindly remove the option for AuthFastLeaderElection (and the class itself) making a note in documentation that it has been deprecated since 3.4.0.

Reply:
done